### PR TITLE
Add new "defined by profile" values for encrypted rtp header extension packets

### PR DESCRIPTION
**Pull request by murillo128**

### CODE REVIEW

File: draft-uberti-avtcore-cryptex.md [hunk 1]
```diff
@@ -141,7 +141,15 @@ Signaling
 
 RTP Header Processing
 =====================
-
+[RFC8285] defines two values for the "defined by profile" field for carrying one byte and two bytes header extensions. In order to allow a receiver to differentiate if an incoming RTP packet is using the encryption scheme  in this specification or the old one, two new values are defined:
+
+ - 0XC0DE for the encrypted version of the one byte header extensions, instead of 0xBEDE
+ - 0XC2DE for the encrypted versions of the two bytes header extensions, instead of 0x100.
```
Comment (JonathanLennox) [inline draft-uberti-avtcore-cryptex.md:147]: You're inconsistent between `0X` and `0x` for hex. I suggest consistently using the latter.

File: draft-uberti-avtcore-cryptex.md [hunk 2]
```diff
@@ -141,7 +141,15 @@ Signaling
 
 RTP Header Processing
 =====================
-
+[RFC8285] defines two values for the "defined by profile" field for carrying one byte and two bytes header extensions. In order to allow a receiver to differentiate if an incoming RTP packet is using the encryption scheme  in this specification or the old one, two new values are defined:
```
Comment (ibc) [inline draft-uberti-avtcore-cryptex.md:144]:
```suggestion
[RFC8285] defines two values for the "defined by profile" field for carrying one byte and two bytes header extensions. In order to allow a receiver to differentiate if an incoming RTP packet is using the encryption scheme in this specification or the old one, two new values are defined:
```

File: draft-uberti-avtcore-cryptex.md [hunk 3]
```diff
@@ -141,7 +141,15 @@ Signaling
...
+
+ - 0xC0DE for the encrypted version of the one byte header extensions, instead of 0xBEDE
```
Comment (ibc) [inline draft-uberti-avtcore-cryptex.md:146]:
```suggestion
 - 0xC0DE for the encrypted version of the one byte header extensions, instead of 0xBEDE.
```

**Comment by ibc** [comment]
Proposal: srtp encrypt/decrypt action should also change the header extension id value, this is, srtp decrypt should rewrite id 0xC0DE to 0xBEDE and vice-versa.

**Comment by murillo128** [comment]
this is already proposed in 

https://github.com/juberti/cryptex/pull/1#discussion_r463658970

I think we should only specify the wire format, my PR to libsrtp will do that, but it's an implementation detail IMO.

**Review by juberti — APPROVED** [review]

**Pull request merged**